maquetacion/scss/titles.scss

```diff
@@ -0,0 +1,12 @@
+@import 'variables';
+
+.title {
+  background-image: url(assets/triangle.svg);
+  background-repeat: no-repeat;
+  font-family: 'Roboto', serif;
+  font-size: $text-big;
+  height: $title-decoration-height;
+  line-height: $title-decoration-height;;
+  text-indent: 50px;
+  margin: $title-decoration-height $title-decoration-width;
```

Comment:
You can add this background element as a before element so you can have better control of it and can be more 'component' based